README.md

```diff
@@ -120,6 +120,20 @@ mod ffi {
 }
 ```
 
+## Comparison to other Rust Qt bindings
```

Comment:
BTW we had come up with this while making a presentation ...

A list of use cases where existing solutions may be suitable
```
I want to use arbitrary Qt API unsafely from Rust
* Ritual
I want to attempt to remove C++ and write everything in Rust
* qmetaobject-rs
I have a small data boundary between Rust and Qt and don’t mind writing binding definitions by hand
* rust-qt-binding-generator
```

And then a list of use cases where CXX-Qt is suited
```
CXX-Qg
* Step-by-step ports to Rust
* Projects with a large amount of QML/C++ knowledge
* Clear data boundary between Rust and Qt
* Integrate C++ and Rust ecosystems
```

Reply:
> I have a small data boundary between Rust and Qt and don’t mind writing binding definitions by hand
* rust-qt-binding-generator

I would recommend cxx-qt for this use case over an unmaintained project.